docs/book/src/reference/webhook-for-core-types.md

```diff
@@ -9,57 +9,49 @@ in controller-runtime.
 It is suggested to use kubebuilder to initialize a project, and then you can
 follow the steps below to add admission webhooks for core types.
 
-## Implement Your Handler
+## Implement Your Webhook
```

Comment:
This example has now changed from a low-level generic webhook to the next-level mutating webhook (`CustomDefaulter`). I think that is good, but I wonder if we could make that clearer when introducing the example? 🤔 Or maybe not - the example seems to still be around adding annotations to pods.

Reply:
I clarified this a bit in the intro section.